files/docker-entrypoint_centos.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 # first arg is `-f` or `--some-option`
 # or first arg is `something.conf`
-if [ "${1:0:1}" = '-' ]; then
+if [ " -${1#?}" = " $1" ]; then
```

Comment:
Could you please explain the reason of this change in the commit message and PR description

Reply:
This fix helps working script in ubuntu and centos.